src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -192,6 +192,9 @@
   <data name="OpenSystemMenuCommandKey" xml:space="preserve">
     <value>Open system menu</value>
   </data>
+  <data name="OriginTagUser" xml:space="preserve">
+    <value>User</value>

```

Comment:
I would not recommend localizing this! The stuff in settings.json is typically untranslated, because the user can move machines/speak multiple languages/change their OS display language and we want everything in that file to parse exactly the same.

This will result in first launch on upgrade generating `User.sendInput.0` in English, and then if the user changes languages and adds additional commands they may get `Usuario.sendInput.0` or `Benutzereinstellungen.sendInput.0`